extensions/applicationinsights-debugplugin-js/scripts/uploadFilesToAzureStorage.ps1

```diff
@@ -29,7 +29,7 @@ switch ($input)
 }
 
 # Add "b/" blob prefix for scripts container only
-$blobPrefix = If ($containerName.equals("scripts")) {"b/"} Else {""}
+$blobPrefix = If ($containerName.equals("scripts")) {"b/ext/"} Else {"ext/"}
```

Comment:
Updating and keeping this as a fallback script that requires and uses the storage account key.